### PR TITLE
Verify The Cut And Dry Pay In Customer Invoices

### PR DESCRIPTION
**Pull request by ashaneo**
This test case is verify the adding new payment method in customer invoices

### CODE REVIEW

File: src/main/java/com/cutanddry/qa/functions/Customer.java [hunk 1]
```diff
@@ -1260,6 +1260,33 @@ public static void editOrderFromReviewScreen() {
...
+        customersPage.selectAccountType(accountType);
+    }
+
+    public static void clickBtnNext(){
+        customersPage.clickNext();
+    }
+
+    public static boolean isPaymentMethodAddedSuccessfully(){
+        return customersPage.isPaymentMethodAddedSuccessfully();
+
```
Comment (AhsanMubarak) [inline src/main/java/com/cutanddry/qa/functions/Customer.java:1289]: Please close the isPaymentMethodAddedSuccessfully() ->}

Comment (ashaneo) [reply src/main/java/com/cutanddry/qa/functions/Customer.java:1290]: Done

File: src/main/java/com/cutanddry/qa/pages/CustomersPage.java [hunk 2]
```diff
@@ -2247,6 +2247,60 @@ public boolean isOrderMiniumErrorBannerDisplayed(){
...
+        distributorUI.clear(tbx_routing_number);
+        distributorUI.sendKeys(tbx_routing_number, routingNumber);
+    }
+
+    public void selectAccountType(String accountType) {
+        try {
+            distributorUI.click(dropDownAccountType);
+            System.out.println("Dropdown opened successfully.");
+
+            By btn_accountType = By.xpath("//div[contains(@class, 'themed_select__option') and text()='" + accountType + "']");
```
Comment (AhsanMubarak) [inline src/main/java/com/cutanddry/qa/pages/CustomersPage.java:2275]: Keep the selectors at the top of the page. You can modify it like this:
Example:
 String lbl_itemAdded = "//div[text()='Top Category Picks']//following-sibling::div//div[text()='CODE']";
 
 Inside the method:
 distributorUI.isDisplayed(By.xpath(lbl_itemAdded.replace("CODE", code)));

**Review by AhsanMubarak — CHANGES_REQUESTED** [review]
Please update the code base based on the review comments and update the PR.

File: src/main/java/com/cutanddry/qa/pages/CustomersPage.java (hunk 2 above)
Comment (ashaneo) [reply src/main/java/com/cutanddry/qa/pages/CustomersPage.java]: Changes

**Review by AhsanMubarak — APPROVED** [review]

**Pull request merged**